lms/static/scripts/frontend_apps/services/client-rpc.ts

```diff
@@ -20,6 +24,12 @@ export type AnnotationEventData = {
   };
 };
 
+/** Argument for `reportUnsavedChanges` message. */
+type UnsavedChangesData = {
+  /** True if the Hypothesis client has unsaved changes to an annotation. */
+  unsaved: boolean;
+};
```

Comment:
Use an object for the message arguments so we can add more fields if needed in future.